src/utils/routing-utils.js

```diff
@@ -0,0 +1,6 @@
+const GARDEN_PATH_REGEX = /(.*?garden\/0x[0-9a-zA-Z]{40}).*/ // TODO: Update when aragonIds supported
+
+export function buildGardenPath(location, relativePath) {
```

Comment:
Because if for example we are in `*/garden/0x../collateral` and we want to redirect to a proposal `*/garden/0x../proposal/0` we can't just push to the current location, we need to remove the replace the relative path with the new one